src/AbstractSQLOptimiser.ts

```diff
@@ -531,6 +531,18 @@ const SubtractDateMatcher = tryMatches<
 	),
 );
 
+const EndsWithMatcher = rewriteMatch(
```

Comment:
I don't see why you need to extract this piece if it is not used anywhere else :thinking:

Reply:
It's used in 2 places and I didn't want to copy/paste it in both